README.md

```diff
@@ -4,59 +4,22 @@
 
 This repository is used to generate the [documentation website for the Tech Docs Template][tdt-docs] and uses the template itself.
 
-The Tech Docs Template is a [middleman template][mmt] that
-you can use to build technical documentation websites using a GOV.UK style.
+The Tech Docs Template is a [middleman template][mmt] that you can use to build technical documentation websites using a GOV.UK style.
```

Comment:
Not sure if this bit is needed as it seems more relevant to the TDT itself (rather than TDT's documentation).